src/lib.rs

```diff
@@ -194,12 +184,7 @@ pub trait Reader: Sized {
     /// Get the nth worksheet. Shortcut for getting the nth
     /// sheet_name, then the corresponding worksheet.
     fn worksheet_range_at(&mut self, n: usize) -> Option<Result<Range<DataType>, Self::Error>> {
-        let name = if let Some(name) = self.sheet_names().get(n) {
-            name
-        } else {
-            return None;
-        }
-        .to_string();
+        let name = self.sheet_names().get(n)?.to_string();
```

Comment:
```suggestion
        let name = self.sheet_names().get(n)?.as_deref();
```

Reply:
Thanks, note this is nightly-only though: https://doc.rust-lang.org/std/result/enum.Result.html#method.as_deref

But so is the use of Bencher at the moment.